app/templates/views/service-settings/request-to-go-live.html

```diff
@@ -2,80 +2,117 @@
-          current_service.has_estimated_usage,
-          'Tell us how many messages you expect to send',
-          url_for('main.estimate_usage', service_id=current_service.id),
-        ) }}
-        {{ task_list_item(
-          current_service.has_team_members_with_manage_service_permission,
-          'Add a team member who can manage settings, team and usage',
-          url_for('main.manage_users', service_id=current_service.id),
-        ) }}
-        {{ task_list_item(
-          current_service.has_templates,
-          'Add templates with examples of the content you plan to send',
-          url_for('main.choose_template', service_id=current_service.id),
-        ) }}
-        {% if current_service.intending_to_send_email %}
-          {{ task_list_item(
-            current_service.has_email_reply_to_address,
-            'Add a reply-to email address',
-            url_for('main.service_email_reply_to', service_id=current_service.id),
-          ) }}
-        {% endif %}
-        {% if (
-          current_service.intending_to_send_sms
-          and current_service.shouldnt_use_govuk_as_sms_sender
-        ) %}
-          {{ task_list_item(
-            not current_service.sms_sender_is_govuk,
-            'Change your Text message sender ID',
-            url_for('main.service_sms_senders', service_id=current_service.id),
-          ) }}
-        {% endif %}
-        {% if current_service.able_to_accept_agreement %}
-          {{ task_list_item(
-            current_service.organisation.agreement_signed,
-            'Accept our data processing and financial agreement',
-            url_for('main.service_agreement', service_id=current_service.id),
-          ) }}
-        {% endif %}
-      {% endcall %}
+      {{ page_header('Make your service live') }}
+      <p class="govuk-body">To remove the <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_trial_mode') }}">trial mode</a> restrictions and make your service live:</p>
+      <ol class="govuk-list govuk-list--number">
+        <li>Complete the tasks on this page.</li>
+        <li>Select ‘Send a request to go live’</li>
+      </ol>
+      <p class="govuk-body">It can take up to one working day to make a service live. This is because we need to check that it meets our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.terms_of_use') }}">terms of use</a>.</p>
+      {# core set of task for to do #}
+      {% set task_list_items = 
+        [{
+          "title": {
+            "text": "Tell us how many messages you expect to send",
+            "classes": "govuk-link--no-visited-state",
+          },
+          "href": url_for('main.estimate_usage', service_id=current_service.id),
+          "status": completed_status if current_service.has_estimated_usage else not_completed_status,
+        },
+        {
+          "title": {
+            "text": "Add a team member who can manage settings, team and usage",
```

Comment:
Probably making this content change now, as we observed some confusion over the current wording.

```suggestion
            "text": "Give another team member the ‘manage settings’ permission",
```